Week-6/Session 15 Js Behind Scenes/s15_mapInEcommerce.js

```diff
@@ -0,0 +1,27 @@
+const productViews = new WeakMap();
+productId = { product1: 1, product2: 2 };
```

Comment:
```suggestion
let productId = { product1: 1, product2: 2 };
```

Reply:
always put var type, missing that can lead to issues, default to using let for vars to be reassigned later, and const for constants.